### ISSUE TITLE
Mails sent by Wekan should have Date header

### ISSUE THREAD

**Issue by furkanmustafa** [wekan/wekan#547]
An example whole source of email, with no Date header.

```
Return-Path: <no-reply@meteor.com>
X-Original-To: furkan@***********
Delivered-To: furkan@***********
Mime-Version: 1.0
From: "Meteor Accounts" <no-reply@meteor.com>
To: furkan@******************************
Subject: Reset your password on rello.***********
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Hello ******************************,

To reset your password, simply click the link below.

https://rello.***********/reset-password/******************************

Thanks.
```

(Off-topic note) Return-Path and From addresses are extreme defaults imho. Would be nice if they're configurable from docker-compose.yml.

**Comment by AlexanderS** [comment]
You can set the environment variable `MAIL_FROM` to change the From address.

**Comment by AlexanderS** [comment]
The Date header topic is an upstream problem of meteor: meteor/meteor#3940

**Comment by AlexanderS** [comment]
This would be fixed by updating meteor to at least [1.3.3](https://github.com/meteor/meteor/blob/master/History.md#v133).

**Comment by AlexanderS** [comment]
Should be fixed with 35c6b51a3f7fe878983620e6755ea35a1cf670c8.
